staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -78,8 +78,8 @@ type Options struct {
 	// See: https://openid.net/specs/openid-connect-core-1_0.html#IDToken
 	ClientID string
 
-	// Path to a PEM encoded root certificate of the provider.
```

Comment:
Do we care that the removal of this field may cause compile failures for downstream consumers of this package?

Reply:
No, these repos offer no [compatibility](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/README.md#compatibility) across versions.